ink-wrapper-types/src/calls.rs

```diff
@@ -84,7 +84,7 @@ impl<T: Send> InstantiateCallNeedsValue<T> {
 
 /// Represents a mutating contract call to be made.
 #[derive(Debug, Clone)]
-pub struct ExecCall {
+pub struct ExecCall<T: scale::Decode + Send> {
```

Comment:
`T` is needed only for decoding result, is that correct? if so, then is it possible to move this generic to the final place, like some retrieving/decoding method?

Reply:
I remove the trait bounds and it seems everything continue to compile...